packages/react-tinacms-inline/src/blocks/inline-block-video.tsx

```diff
@@ -0,0 +1,27 @@
+/**
+
+Copyright 2019 Forestry.io Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+import * as React from 'react'
+import { InlineVideoProps, InlineVideo } from '../fields/inline-video'
+
+/**
+ * @deprecated
```

Comment:
This looks like a copy / paste of other deprecated block field files. We can delete this file and just recommend folks use `InlineVideo` from the start.